projects/gloo/api/v1/enterprise/options/ai/ai.proto

```diff
@@ -9,714 +9,738 @@ option (extproto.equal_all) = true;
+//
+// To get started, see [About Gloo AI Gateway]({{% versioned_link_path fromRoot="/ai/overview/" %}}).
+// For more information about the Upstream resource, see the
+// [API reference]({{% versioned_link_path fromRoot="/reference/api/github.com/solo-io/gloo/projects/gloo/api/v1/upstream.proto.sk/" %}}).
+//
+// {{% notice note %}}
+// AI Gateway is an Enterprise-only feature that requires a Gloo Gateway Enterprise license with an AI Gateway add-on.
+// {{% /notice %}}
 message UpstreamSpec {
 
-  // Settings to configure a custom host to send the traffic to
+  // Send requests to a custom host and port, such as to proxy the request,
+  // or to use a different backend that is API-compliant with the upstream version.
   message CustomHost {
-    // Custom host to send the traffic to
+    // Custom host to send the traffic requests to.
     string host = 1;
-    // Custom port to send the traffic to
+    // Custom port to send the traffic requests to.
     uint32 port = 2;
   }
 
-  // Settings for the OpenAI API
+  // Settings for the [OpenAI](https://platform.openai.com/docs/overview) LLM provider.
   message OpenAI {
-    // Auth Token to use for the OpenAI API
-    // This token will be placed into the `Authorization` header
-    // and prefixed with Bearer if not present
-    // when sending the request to the upstream.
+    // The authorization token that the AI gateway uses to access the OpenAI API.
+    // This token is automatically sent in the `Authorization` header of the
+    // request and prefixed with `Bearer`.
     SingleAuthToken auth_token = 1;
-    // Optional custom host to send the traffic to
+    // Optional: Send requests to a custom host and port, such as to proxy the request,
+    // or to use a different backend that is API-compliant with the upstream version.
     CustomHost custom_host = 2;
-    // Optional: override model name. If not set, the model name will be taken from the request
-    // This can be useful when trying model failover scenarios
-    // e.g. "gpt-4o-mini"
+    // Optional: Override the model name, such as `gpt-4o-mini`.
+    // If unset, the model name is taken from the request.
+    // This setting can be useful when testing model failover scenarios.
     string model = 3;
   }
 
-  // Settings for the Azure OpenAI API
+  // Settings for the [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/) LLM provider.
+  // To find the values for the endpoint, deployment name, and API version, you can check the fields of an API request, such as
+  // `https://{endpoint}/openai/deployments/{deployment_name}/chat/completions?api-version={api_version}`.
   message AzureOpenAI {
-    // Auth Token to use for the Azure OpenAI API
-    // This token will be placed into the `api-key` header
+    // The authorization token that the AI gateway uses to access the Azure OpenAI API.
+    // This token is automatically sent in the `api-key` header of the request.
     oneof auth_token_source {
```

Comment:
To be completely frank I don't remember my exact intention here. I vaguely remember wanting to separate situations where we fetched a token for the user, vs allowing them to provide one, but I don't seem to have followed through there with the `Passthrough` field, so it may be worth revisiting this decision